x/oracle/types/plan.go

```diff
@@ -0,0 +1,106 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (p ParamUpdatePlan) String() string {
+	due := p.DueAt()
+	return fmt.Sprintf(`Oracle Param Update Plan
+  Keys: %s
+  %s
+  Changes: %s.`, p.Keys, due, p.Changes)
+}
+
+// ValidateBasic does basic validation of a ParamUpdatePlan
+func (p ParamUpdatePlan) ValidateBasic() error {
```

Comment:
I'm not sure how we run into this here since this ValidateBasic() function only validates that the AcceptList and Mandatory list are valid denom lists (not if they're valid between each other). The other validation method in the keeper [ValidateParamChanges](https://github.com/ojo-network/ojo/blob/2da97db9297c67b804fdc530612e0409b33ec4df/x/oracle/keeper/keeper.go#L401) is the one that calls the method [Validate](https://github.com/ojo-network/ojo/blob/2da97db9297c67b804fdc530612e0409b33ec4df/x/oracle/types/params.go#L198) which does the validation between all params. However this is done after checking all the param changes first in the method so it still shouldn't hit this case.